documentation/translations/translating.rst

```diff
@@ -39,9 +39,9 @@ For more details about translations and their progress, see `the dashboard
        | Fanis Petkos (:github-user:`thepetk`),
        | Panagiotis Skias (:github-user:`skpanagiotis`)
      - :github:`GitHub <python/python-docs-el>`
-   * - Hindi (hi-IN)
-     - Sanyam Khurana (:github-user:`CuriousLearner`)
-     - :github:`GitHub <CuriousLearner/python-docs-hi-in>`
+   * - Hindi (hi)
+     - Sanyam Khurana (:github-user:`himanshugarg`)
```

Comment:
Is there a specific reason to change from `hi-IN` to `hi`?

Please also update your full name, Sanyam Khurana (@CuriousLearner) is still listed.

Reply:
Will revert the code and fix the name. Thanks